.github/workflows/python-package.yaml

```
@@ -0,0 +1,79 @@
+name: Test
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ master ]
```

I would just simplify to

```suggestion
on: [push, pull_request]
```